lua/nvim-mapper.lua

```diff
@@ -3,101 +3,115 @@ local M = {}
 M.mapper_records = {}
 
 -- Set a mapping
-local function map(virtual, buffnr, mode, keys, cmd, options, category,
-                   unique_identifier, description)
+local function map(virtual, buffnr, mode, keys, cmd, options, category, unique_identifier, description)
+  local buffer_only
+  if buffnr == nil then
+    buffer_only = false
+  else
+    buffer_only = true
+  end
 
-    local buffer_only
-    if buffnr == nil then
-        buffer_only = false
-    else
-        buffer_only = true
-    end
+  local record = {
+    mode = mode,
+    keys = keys,
+    cmd = cmd,
+    options = options,
+    category = category,
+    unique_identifier = unique_identifier,
+    description = description,
+    buffer_only = buffer_only,
+  }
 
-    local record = {
-        mode = mode,
-        keys = keys,
-        cmd = cmd,
-        options = options,
-        category = category,
-        unique_identifier = unique_identifier,
-        description = description,
-        buffer_only = buffer_only
-    }
+  maybe_existing_record = M.mapper_records[unique_identifier]
 
-    maybe_existing_record = M.mapper_records[unique_identifier]
+  if maybe_existing_record == nil then
+    M.mapper_records[unique_identifier] = record
+  elseif maybe_existing_record.mode ~= mode
+      or maybe_existing_record.keys ~= keys
+      or maybe_existing_record.cmd ~= cmd
+      or maybe_existing_record.category ~= category
+      or maybe_existing_record.description ~= description
+  then
+    print("Mapper error : unique identifier " .. unique_identifier .. " cannot be used twice")
+  end
 
-    if maybe_existing_record == nil then
-        M.mapper_records[unique_identifier] = record
-    elseif (maybe_existing_record.mode ~= mode or
-            maybe_existing_record.keys ~= keys or
-            maybe_existing_record.cmd ~= cmd or
-            maybe_existing_record.category ~= category or
-            maybe_existing_record.description ~= description) then
-        print("Mapper error : unique identifier " .. unique_identifier ..
-              " cannot be used twice")
-    end
-
-    -- Set the mapping
-    if not virtual then
-        if buffnr ~= nil then
-            vim.api.nvim_buf_set_keymap(buffnr, mode, keys, cmd, options)
-        else
-            vim.api.nvim_set_keymap(mode, keys, cmd, options)
-        end
+  -- Set the mapping
```

Comment:
Most of this file is just sumneko_lua LSP formatting. This block is the only one that has actual code changes